src/js/utils/ServiceUtil.js

```diff
@@ -54,6 +75,12 @@ const ServiceUtil = {
     appDefinition.instances = service.getInstancesCount();
     appDefinition.cmd = service.getCommand();
 
```

Comment:
Maybe I should add a comment here I thought it would be handy to have a section per section lets say

```JS
// General
definition.id = service.getID();
...
...

// Optional
definition.executor = service.getExecutor();
...
...